### ISSUE TITLE
groot binaries check build stamp before downloading

### ISSUE THREAD

**Issue by stonier** [yujinrobot/yujin_tools#95]
Much faster than waiting for the rsync to traverse the whole world of groot binaries.

**Comment by asmodehn** [comment]
This recent feature seems to not report failure correctly : 
```
TASK [yujinrobot.ros : Update all ros-indigo packages] *************************
ok: [192.168.3.93]

TASK [yujinrobot.ros : RosDep init] ********************************************
changed: [192.168.3.93]

TASK [yujinrobot.ros : Rosdep update (warning: changes undetectable)] **********
ok: [192.168.3.93]

TASK [yujinrobot.gopher-software : Ensure /opt/yujin/amd64 directory exists and is accessible by yujin.] ***
changed: [192.168.3.93]

TASK [yujinrobot.gopher-software : Sync'ing groot binaries (internal)(stable)] *
ok: [192.168.3.93]

TASK [yujinrobot.gopher-software : Install rosdeps] ****************************
skipping: [192.168.3.93]

TASK [yujinrobot.gopher-software : Downloading default map (heaven)] ***********
fatal: [192.168.3.93]: FAILED! => {"changed": true, "cmd": "source /opt/yujin/amd64/indigo-stable/setup.bash && /opt/yujin/amd64/indigo-stable/bin/gopher_maps -d heaven --non-interactive", "delta": "0:00:00.002047", "end": "2016-08-26 14:04:06.687169", "failed": true, "rc": 1, "start": "2016-08-26 14:04:06.685122", "stderr": "/bin/bash: /opt/yujin/amd64/indigo-stable/setup.bash: No such file or directory", "stdout": "", "stdout_lines": [], "warnings": []}

NO MORE HOSTS LEFT *************************************************************
 [WARNING]: Could not create retry file 'concert-ros_concert.retry'.         [Errno 2] No such file or directory: ''
```

More directly : 
```
(yujin-ansible)yujin@concert:~/ansible_playbooks$ groot_binaries --stable
ssh: Could not resolve hostname files.yujin.com: Name or service not known
Nothing to do: remote and local builds have the same timestamps.
(yujin-ansible)yujin@concert:~/ansible_playbooks$ echo $?
0
```

**Comment by stonier** [comment]
> ssh: Could not resolve hostname files.yujin.com: Name or service not known
> Nothing to do: remote and local builds have the same timestamps

So something wrong with the check there. Thanks for the heads up.

**Comment by stonier** [comment]
Released in 0.4.35.
